modules/claims_api/spec/lib/claims_api/vnp_ptcpnt_addrs_service_spec.rb

```diff
@@ -0,0 +1,78 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+require 'bgs_service/vnp_ptcpnt_addrs_service'
+
+describe ClaimsApi::VnpPtcpntAddrsService do
+  subject { described_class.new external_uid: 'xUid', external_key: 'xKey' }
+
+  describe 'vnp_ptcpnt_addrs_create' do
+    let(:options) { {} }
+
+    it 'responds with attributes' do
+      options[:vnp_ptcpnt_addrs_id] = nil
+      options[:vnp_proc_id] = '3854596'
+      options[:vnp_ptcpnt_id] = '182057'
+      options[:efctv_dt] = '2020-07-16T18:20:18Z'
+      options[:addrs_one_txt] = '76 Crowther Ave'
+      options[:addrs_three_txt] = nil
+      options[:addrs_two_txt] = nil
+      options[:bad_addrs_ind] = nil
+      options[:city_nm] = 'Bridgeport'
+      options[:cntry_nm] = nil
+      options[:county_nm] = nil
+      options[:eft_waiver_type_nm] = nil
+      options[:email_addrs_txt] = 'testy@test.com'
+      options[:end_dt] = nil
+      options[:fms_addrs_code_txt] = nil
+      options[:frgn_postal_cd] = nil
+      options[:group_1_verifd_type_cd] = nil
+      options[:jrn_dt] = '2020-07-16T18:20:17Z'
+      options[:jrn_lctn_id] = 281
+      options[:jrn_obj_id] = 'VAgovAPI'
+      options[:jrn_status_type_cd] = 'U'
+      options[:jrn_user_id] = 'VAgovAPI'
+      options[:lctn_nm] = nil
+      options[:mlty_postal_type_cd] = nil
+      options[:mlty_post_office_type_cd] = nil
+      options[:postal_cd] = 'CT'
+      options[:prvnc_nm] = 'CT'
+      options[:ptcpnt_addrs_type_nm] = 'Mailing'
+      options[:shared_addrs_ind] = 'N'
+      options[:trsury_addrs_five_txt] = nil
+      options[:trsury_addrs_four_txt] = nil
+      options[:trsury_addrs_one_txt] = nil
+      options[:trsury_addrs_six_txt] = nil
+      options[:trsury_addrs_three_txt] = nil
+      options[:trsury_addrs_two_txt] = nil
+      options[:trsury_seq_nbr] = nil
+      options[:trtry_nm] = nil
+      options[:zip_first_suffix_nbr] = nil
+      options[:zip_prefix_nbr] = '06605'
+      options[:zip_second_suffix_nbr] = nil
+
+      VCR.use_cassette('bgs/vnp_ptcpnt_addrs_service/vnp_ptcpnt_addrs_create') do
```

Comment:
we may want to merge and use the new recording setup to keep in line with our new BGS service specs. it's also semi-dependent on [these changes](https://github.com/department-of-veterans-affairs/vets-api/pull/16232), so could wait for that to merge before doing so.

Reply:
Good point. I'll ask someone for a brief hands-on safety check with that big PR at the beginning of tomorrow so we feel good about merging it. The PR already has the necessary approvals and passes all checks so we can get it merged in quickly. Then we can slightly restructure this one to match the new BGS unit test pattern which I'm free to help demo at any time if you would like @mchristiansonVA or you can copy off of one of the two new `ManageRepresentativeService` specs.